tests/integration/suite/test_cluster_scan.py

```diff
@@ -0,0 +1,20 @@
+from .common import random_str
+from .conftest import cluster_and_client
+
+
+def test_run_scan_not_available_on_not_ready_cluster(admin_mc,
+                                                     remove_resource):
+    client = admin_mc.client
+    cluster = client.create_cluster(
+        name=random_str(),
+        rancherKubernetesEngineConfig={
+            "accessKey": "junk"
+        }
+    )
+    remove_resource(cluster)
+    _, cluster_client = cluster_and_client(cluster.id, client)
+    try:
+        admin_mc.client.action(obj=cluster, action_name="runSecurityScan", )
+    except AttributeError as e:
+        assert e is not None
```

Comment:
This check should be specific to the error you are expecting, code and message. Update it to use pytest.raises, as seen [here](https://github.com/rancher/rancher/blob/master/tests/integration/suite/test_app.py#L366)

Reply:
@dramich The action is not available, hence the error is "AttributeError". The check that you have pointed will work when the action returns an actual error code/message. Nothing to be changed here.

Reply:
Ah, ok, then it should be checking the links to validate it doesn't exist

Reply:
I guess you meant actions, not links. Modified, please re-review.